src/lib/dialog_ui.sh

```diff
@@ -373,6 +373,61 @@ function create_directory_selection_screen()
   return "$ret"
 }
 
+# Create file selection screen.
+#
+# @starting_path: Path that is in the input box when screen starts
+# @box_title: Title of the box
+# @height: Menu height in lines size
+# @width: Menu width in column size
+# @flag How to display a command, the default value is
+#   "SILENT". For more options see `src/kwlib.sh` function `cmd_manager`
+#
+# Return:
+# Returns 0 if the 'OK' button is pressed, 1 if the 'Cancel' button is pressed
+# and 2 if the 'Help' button is pressed. The string in the input box (current
+# path) is returned via the `menu_return_string` variable.
+function create_file_selection_screen()
+{
+  local starting_path="$1"
+  local box_title="$2"
+  local height="$3"
+  local width="$4"
+  local flag="$5"
```

Comment:
fselect does not support an extra button and label rename? If so, add those extra parameters.